website/content/api-docs/query.mdx

```diff
@@ -194,14 +194,25 @@ The table below shows this endpoint's support for
       determine the priority.
 
     - `Datacenters` `(array<string>: nil)` - Specifies a fixed list of remote
-      datacenters to forward the query to if there are no healthy nodes in the
+      datacenters to forward the query to when there are no healthy nodes in the
       local datacenter. Datacenters are queried in the order given in the
       list. If this option is combined with `NearestN`, then the `NearestN`
       queries will be performed first, followed by the list given by
       `Datacenters`. A given datacenter will only be queried one time during a
       failover, even if it is selected by both `NearestN` and is listed in
       `Datacenters`.
 
+    - `Targets` `(array<Target>: nil)` - Specifies a fixed list of remote
+      datacenters and cluster peers to failover to  if there are no healthy
+      nodes in the local datacenter. Queries targets in the order listed.
+      This option cannot be used with `NearestN` or `Datacenters`.
+
+      - `PeerName` `(string: "")` - Specifies a cluster peer to use for
```

Comment:
Consider linking to the cluster peering docs (in case someone doesn't know what that is).